<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>525</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>38</red>
        <green>39</green>
        <blue>40</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>2</red>
        <green>38</green>
        <blue>9</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>38</red>
        <green>39</green>
        <blue>40</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>2</red>
        <green>38</green>
        <blue>9</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>2</red>
        <green>38</green>
        <blue>9</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>2</red>
        <green>38</green>
        <blue>9</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>SoCraps</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>411</width>
      <height>281</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Place Bets</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QDoubleSpinBox" name="fourSpinBox">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>20</y>
       <width>91</width>
       <height>25</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="prefix">
      <string>4 - $</string>
     </property>
     <property name="suffix">
      <string/>
     </property>
     <property name="decimals">
      <number>0</number>
     </property>
     <property name="maximum">
      <double>1000000.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="fiveSpinBox">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>40</y>
       <width>91</width>
       <height>25</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="prefix">
      <string>5 - $</string>
     </property>
     <property name="suffix">
      <string/>
     </property>
     <property name="decimals">
      <number>0</number>
     </property>
     <property name="maximum">
      <double>1000000.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="sixSpinBox">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>60</y>
       <width>91</width>
       <height>25</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="prefix">
      <string>6 - $</string>
     </property>
     <property name="suffix">
      <string/>
     </property>
     <property name="decimals">
      <number>0</number>
     </property>
     <property name="maximum">
      <double>1000000.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="eightSpinBox">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>80</y>
       <width>91</width>
       <height>25</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="prefix">
      <string>8 - $</string>
     </property>
     <property name="suffix">
      <string/>
     </property>
     <property name="decimals">
      <number>0</number>
     </property>
     <property name="maximum">
      <double>1000000.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="nineSpinBox">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>100</y>
       <width>91</width>
       <height>25</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="prefix">
      <string>9 - $</string>
     </property>
     <property name="suffix">
      <string/>
     </property>
     <property name="decimals">
      <number>0</number>
     </property>
     <property name="maximum">
      <double>1000000.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="tenSpinBox">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>120</y>
       <width>91</width>
       <height>25</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="prefix">
      <string>10 - $</string>
     </property>
     <property name="suffix">
      <string/>
     </property>
     <property name="decimals">
      <number>0</number>
     </property>
     <property name="maximum">
      <double>1000000.000000000000000</double>
     </property>
    </widget>
    <widget class="QSpinBox" name="RBetSpinBox">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>250</y>
       <width>46</width>
       <height>25</height>
      </rect>
     </property>
     <property name="minimum">
      <number>2</number>
     </property>
     <property name="maximum">
      <number>12</number>
     </property>
     <property name="value">
      <number>7</number>
     </property>
    </widget>
    <widget class="QPushButton" name="rBetButton">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>250</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Remove Bet</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="amountSpinBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>250</y>
       <width>91</width>
       <height>25</height>
      </rect>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="prefix">
      <string>$</string>
     </property>
     <property name="suffix">
      <string/>
     </property>
     <property name="decimals">
      <number>0</number>
     </property>
     <property name="maximum">
      <double>1000000.000000000000000</double>
     </property>
     <property name="value">
      <double>0.000000000000000</double>
     </property>
    </widget>
    <widget class="QPushButton" name="placeBetButton">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>250</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Place Bet</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="betSpinBox">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>250</y>
       <width>46</width>
       <height>25</height>
      </rect>
     </property>
     <property name="minimum">
      <number>2</number>
     </property>
     <property name="maximum">
      <number>12</number>
     </property>
     <property name="value">
      <number>7</number>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>220</y>
       <width>79</width>
       <height>24</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Pass</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Don't Pass</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="betPushButton">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>220</y>
       <width>41</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Bet</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="passSpinBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>91</width>
       <height>25</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="prefix">
      <string>Pass - $</string>
     </property>
     <property name="suffix">
      <string/>
     </property>
     <property name="decimals">
      <number>0</number>
     </property>
     <property name="maximum">
      <double>1000000.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="amountSpinBox_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>91</width>
       <height>25</height>
      </rect>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="prefix">
      <string>$</string>
     </property>
     <property name="suffix">
      <string/>
     </property>
     <property name="decimals">
      <number>0</number>
     </property>
     <property name="maximum">
      <double>1000000.000000000000000</double>
     </property>
     <property name="value">
      <double>0.000000000000000</double>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>146</width>
       <height>24</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QRadioButton" name="pointButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Point On:</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="pointLabel">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QDoubleSpinBox" name="dontSpinBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>91</width>
       <height>25</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="prefix">
      <string>Don't - $</string>
     </property>
     <property name="suffix">
      <string/>
     </property>
     <property name="decimals">
      <number>0</number>
     </property>
     <property name="maximum">
      <double>1000000.000000000000000</double>
     </property>
    </widget>
    <widget class="QPushButton" name="repeatButton">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>160</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Repeat Bets</string>
     </property>
    </widget>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_3">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>20</y>
      <width>131</width>
      <height>111</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>320</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>411</width>
      <height>174</height>
     </rect>
    </property>
    <property name="title">
     <string>One Time Bets</string>
    </property>
    <widget class="QDoubleSpinBox" name="threeSpinBox">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>40</y>
       <width>91</width>
       <height>25</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="prefix">
      <string>3 - $</string>
     </property>
     <property name="suffix">
      <string/>
     </property>
     <property name="decimals">
      <number>0</number>
     </property>
     <property name="maximum">
      <double>1000000.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="twoSpinBox">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>20</y>
       <width>91</width>
       <height>25</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="prefix">
      <string>2 - $</string>
     </property>
     <property name="suffix">
      <string/>
     </property>
     <property name="decimals">
      <number>0</number>
     </property>
     <property name="maximum">
      <double>1000000.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="twelveSpinBox">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>100</y>
       <width>91</width>
       <height>25</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="prefix">
      <string>12 - $</string>
     </property>
     <property name="suffix">
      <string/>
     </property>
     <property name="decimals">
      <number>0</number>
     </property>
     <property name="maximum">
      <double>1000000.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="elevenSpinBox">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>80</y>
       <width>91</width>
       <height>25</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="prefix">
      <string>11 - $</string>
     </property>
     <property name="suffix">
      <string/>
     </property>
     <property name="decimals">
      <number>0</number>
     </property>
     <property name="maximum">
      <double>1000000.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="sevenSpinBox">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>60</y>
       <width>91</width>
       <height>25</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="prefix">
      <string>7 - $</string>
     </property>
     <property name="suffix">
      <string/>
     </property>
     <property name="decimals">
      <number>0</number>
     </property>
     <property name="maximum">
      <double>1000000.000000000000000</double>
     </property>
    </widget>
    <widget class="QPushButton" name="oRepeatButton">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>140</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Repeat Bets</string>
     </property>
    </widget>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>450</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="prefix">
     <string>$</string>
    </property>
    <property name="suffix">
     <string/>
    </property>
    <property name="decimals">
     <number>0</number>
    </property>
    <property name="maximum">
     <double>1000000.000000000000000</double>
    </property>
    <property name="value">
     <double>0.000000000000000</double>
    </property>
   </widget>
   <widget class="QPushButton" name="rollButton">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>180</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Roll</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>140</y>
      <width>136</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLCDNumber" name="lcdNumber_2"/>
     </item>
     <item>
      <widget class="QLCDNumber" name="lcdNumber"/>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>400</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Funds:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="depositButton">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>450</y>
      <width>71</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Deposit</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="fundsSpinBox">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>400</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="prefix">
     <string>$</string>
    </property>
    <property name="suffix">
     <string/>
    </property>
    <property name="decimals">
     <number>0</number>
    </property>
    <property name="maximum">
     <double>1000000.000000000000000</double>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
